src/OrbitGl/TrackContainer.h

```diff
@@ -58,9 +58,12 @@ class TrackContainer final : public CaptureViewElement {
           iterator_timer_info,
       const absl::flat_hash_map<uint64_t, uint64_t>& iterator_id_to_function_id);
   void UpdateVerticalScrollUsingRatio(float ratio);
-  void IncrementVerticalScroll(float delta_y);
   [[nodiscard]] float GetVerticalScrollingOffset() const { return vertical_scrolling_offset_; }
   void SetVerticalScrollingOffset(float value);
+  void OnVerticalScroll(int delta);
+  const float kHeightRatioPerPageUpAndDown = 0.9f;
+  void OnPageUp() { IncrementVerticalScroll(kHeightRatioPerPageUpAndDown * GetHeight()); }
+  void OnPageDown() { IncrementVerticalScroll(-kHeightRatioPerPageUpAndDown * GetHeight()); }
```

Comment:
This is a bit of a nitpick, but I find this inconsistent - we're defining the amount of panning for the left / right arrow keys in CaptureWindow, but provide specialized methods for up / down. I think at this stage I would prefer to get rid of the `OnPageUp`/ `OnPageDown` methods and use `IncrementVerticalScroll` directly in `CaptureWindow`. This makes the "how far do we scroll" logic more centralized. But I'll leave this up to you.

Reply:
No, we are not. Everything is managed here in the container. What do you think if I split OnVerticalScroll to OnVerticalScrollUp and OnVerticalScrollDown?

Reply:
You are right. I modified everything to call an unique function in TrackContainer that only takes the ratio. So, now it works similarly to Panning.